pydis_site/apps/api/viewsets/bot/user.py

```diff
@@ -56,6 +104,7 @@ class UserViewSet(BulkCreateModelMixin, ModelViewSet):
     ### POST /bot/users
     Adds a single or multiple new users.
     The roles attached to the user(s) must be roles known by the site.
+    User creation process will be skipped if user is already present in the database.
```

Comment:
The way you worded it is unclear in the sense that it may imply the entire operation will abort.
```suggestion
    Users that already exist in the database will be skipped.
```